tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1753,11 +1753,29 @@ def window(self, size, shift=None, stride=1, drop_remainder=False):
 
     A "window" is a finite dataset of flat elements of size `size` (or possibly
     fewer if there are not enough input elements to fill the window and
-    `drop_remainder` evaluates to false).
+    `drop_remainder` evaluates to `False`).
+
+    The `shift` argument determines the number of input elements by which
+    the window moves on each iteration.  The first element in the `k`th window
+    will be element
+
+    ```
+    1 + (k-1) * shift
+    ```
+
+    of the input dataset. In particular, the first element of the first window
+    will always be the first element of the input dataset.
+
+    If the `stride` parameter is greater than 1, then each window will skip
+    `(stride - 1)` input elements between each element that appears in the
+    window. Output windows will still contain `size` elements regardless of
+    the value of `stride`.
 
     The `stride` argument determines the stride of the input elements, and the
```

Comment:
I think we can remove this sentence now, since there are more detailed explanations before

Reply:
Deleted this paragraph.